### ISSUE TITLE
Provide endpoints for monitoring (data and graphs)

### ISSUE THREAD

**Issue by jbonnet** [sonata-nfv/tng-api-gtw#182]
http://pre-int-sp-ath.5gtango.eu:3000
Or exactly to http://pre-int-sp-ath.5gtango.eu:3000/d-solo/sp_dash/sp?orgId=1&panelId=13&from=58723465723546&......

* GET .../monitoring/graphs
* GET .../monitoring/data

**Comment by anapolg** [comment]
Please expose ` http://pre-int-sp-ath.5gtango.eu:3000 ` as ..../monitoring/graphs
Everything else will be handled with url parameters such as ?panelId=Y?orgId=X....

cc @pkarkazis

**Comment by jbonnet** [comment]
@pkarkazis is `son-monitor` still the name of the monitoring container? If so, the above should be `http://son-monitor:3000`, right?

**Comment by pkarkazis** [comment]
Hi @jbonnet,
regarding the /monitoring/data we must expose the following apis:
GET pre-int-sp-ath.5gtango.eu:8000/api/v2/services/{srv_id}/metrics
GET pre-int-sp-ath.5gtango.eu:8000/api/v2/vnfs/{vnf_id}/metrics
GET pre-int-sp-ath.5gtango.eu:8000/api/v2/vnfs/{vnf_id}/vdu/{vdu_id}/metrics
or we could expose all GET the apis under api/v2/ 
what do you think?

**Comment by pkarkazis** [comment]
@jbonnet the graphs are provided from a separate container son-monitor-grafana:3000

**Comment by jbonnet** [comment]
Related to sonata-nfv/tng-gtk-sp#275

**Comment by jbonnet** [comment]
@pkarkazis @anapolg Can you please provide me with:

1. roles that can access this data: <anonynous>, adminm customer and developer (my guess: only admins and developers)
1. what they can to: get,delete,options,patch,post (my guess developers can only get and options, admins can do all)

Thanks

**Comment by jbonnet** [comment]
@anapolg @pkarkazis Implemented in PR #183, please test and let me know

**Comment by anapolg** [comment]
For the moment I just need GET to get the graphs already provided by @pkarkazis 

Also for the moment anyone can access the dashboard so anyone should be accepted to see this. It will depend on who you consider that must have access to the Dashboard.

**Comment by jbonnet** [comment]
@pkarkazis if there's no need for different validation in each one of them, no problem in exposing all at the `son-monitor-manager:8000/api/v2/`. Is `son-monitor-manager` the name of the container?

**Comment by pkarkazis** [comment]
@jbonnet Yes, the 'son-monitor-manager' is the name of the monitoring manager container
